### PR TITLE
Dr/deploy

### PR DESCRIPTION
**Pull request by diegoribero**
- printJson function added to helpers
- Policies on install and deploy are now pretty printed
- Install refactored
- Deploy now triggers an install

(Requires **matrix-firebase** updated with appstore listeners)

### CODE REVIEW

File: bin/matrix-deploy.js [hunk 1]
```diff
@@ -205,24 +210,50 @@ Matrix.localization.init(Matrix.localesFolder, Matrix.config.locale, function ()
...
-                      
-                      Matrix.helpers.installApp(options, function () { 
-                        endIt();
+                      /*Matrix.firebase.appstore.watchForAppUpdated(appName, function (app) {
+                        debug('App updated > ' + JSON.stringify(app));
+                        setTimeout(function () {
+                          process.nextTick(function () {
+                            triggerInstall(app);
+                          });
+                        }, 500);
```
Comment (brianofrokk3r) [inline bin/matrix-deploy.js]: @diegoribero need these comments?

Comment (diegoribero) [inline bin/matrix-deploy.js]: I'm going to say nay, it's easy to implement if we ever need that. Getting rid of them

File: lib/helpers.js [hunk 2]
```diff
@@ -368,11 +368,10 @@ function checkPolicy(policy, name, cb) {
     cb(null, confirmed);
   });
 
-
   function stepThroughConfig() {
     console.log('config to strep trough: ', JSON.stringify(policy));
     _.forIn(policy, function (v, k) {
-      console.log('>>>>>>> ' , v , ' : ' , k , ' <<<<<<<< ');
+      console.log('>>>>>>> ', v, ' : ', k, ' <<<<<<<< ');
```
Comment (eighteyes) [inline lib/helpers.js]: Please move all these console.logs into debugs. They mess with the install policy generation UX.

**Comment by eighteyes** [comment]
Might want to look at `util.inspect(object[, options])` in node. I'm just saying. It pretty prints for me all the time, and doesn't complain. :)

**Comment by eighteyes** [comment]
When I try to deploy the base object I get

```
Please specify your application access permissions:
{
  'sensors' : {
    'false' : true
  },
  'integrations' : {
    'false' : true
  },
  'services' : {
    'false' : true
  }
}
````
if the value is false, don't set it to true, just skip it.

**Comment by eighteyes** [comment]
This installed to the device, but didn't watch for the install and exit cleanly.

File: lib/helpers.js (hunk 2 above)
Comment (diegoribero) [inline lib/helpers.js]: Done :)

**Comment by diegoribero** [comment]
Haven't been able to verify full flow yet but policies look good now and the install not triggering on device might have been caused by a mix up in DB

**Comment by eighteyes** [comment]
Looks good! Ty!

**Pull request merged**